src/test/java/tests/testng/CartTest.java

```diff
@@ -0,0 +1,70 @@
+package tests.testNg;
+
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+import tests.testNg.retry.MyRetry;
+
+import static tests.testNg.JsonParserTest.carts;
+
+public class CartTest {
+    @Parameters({"cart-number"})
+    @Test(groups = {"goodTests"}, testName = "Check addVirtualItem method")
+    public void checkAddVirtualItem(String cartNumber){
+        double originalPrice = carts.get(Integer.valueOf(cartNumber)).getTotalPrice();
```

Comment:
u can simple use int parameter, testng will convert it for you

Reply:
fixed